### PR TITLE
Add juniper.bot to presences store

### PR DESCRIPTION
**Pull request by MrLivixx**
Adding the juniperbot site to the presences store, more in pl #1940 I was told to fix everything and open it again

### CODE REVIEW

**Comment by MrLivixx** [comment]
I fixed deepscan issues

File: websites/J/Juniperbot/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "url": [
+    "docs.juniper.bot",
+    "feedback.juniper.bot",
+    "juniper.bot"
+  ],
+  "description": {
+    "ru": "Многофункциональный, полностью настраиваемый бот для платформы Discord, с панелью управления.",
+    "en": "Fluffy Discord Bot (Music, Ranking, Reminder, Vk Community bridge, WikiFur)"
+  },
+  "service": "juniperbot",
```
Comment (Bas950) [inline websites/J/Juniperbot/dist/metadata.json:16]: service name needs to be the same as folder name (capitalization included)

File: websites/J/Juniperbot/presence.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+const 
+  presence = new Presence({clientId: "739908991274057870"}),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {largeImageKey: "logo"},
+    juniper: boolean = await presence.getSetting("juniper"),
+    docs: boolean = await presence.getSetting("docs"),
+    fback: boolean = await presence.getSetting("fback");
```
Comment (Bas950) [inline websites/J/Juniperbot/presence.ts:9]: add defaults:
variable: boolean = await presence.getSetting("var") || true

This is needed due to a bug we currently have.

File: websites/J/Juniperbot/presence.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      presenceData.details = "Looks at the list of commands"; 
+      presenceData.smallImageKey = "list";
+    } else if (document.location.pathname == "/status"){
+      presenceData.details = "Looks at the bot's statistics";
+      presenceData.smallImageKey = "stats";
+    } else if (document.location.pathname == "/user/card"){
+      presenceData.details = "Changes the rank card";
+    }
+    }
+    }
```
Comment (Bas950) [inline websites/J/Juniperbot/presence.ts:39]: Is this correct? LOL
If you use VSCode please add the Prettier Extension!

**Comment by MrLivixx** [comment]
> update your fork, then fix any deepscan issues if there still are any

@Bas950,DeepScan swears because I wrote | / true when I get the settings as you asked me. And also there are not my mistakes yet. I updated the fork

**Comment by Bas950** [comment]
> > update your fork, then fix any deepscan issues if there still are any
> 
> @Bas950,DeepScan swears because I wrote | / true when I get the settings as you asked me. And also there are not my mistakes yet. I updated the fork

I think you F'ed up your update fork commit
image
And it says you are changing 21 files now...

**Comment by MrLivixx** [comment]
What should I do?

**Comment by doomlerd** [comment]
> What should I do?

I suggest copying the juniper.bot files, deleting your fork and making another one then applying the changes you copied earlier to it.
I'll be closing the PR now, make another one when you fix the repo's diff, apologies.

**Pull request closed without merging**